modules/util/reflect.go

```diff
@@ -0,0 +1,15 @@
+package util
+
+import (
+	"fmt"
+	"reflect"
+	"runtime"
+)
+
+// GetFunctionInfo is for debug purpose (ex: show a router handler's real name)
+func GetFunctionInfo(i interface{}) string {
+	fp := reflect.ValueOf(i).Pointer()
+	fi := runtime.FuncForPC(fp)
+	file, line := fi.FileLine(fp)
+	return fmt.Sprintf("%s(%s:%d)", fi.Name(), file, line)
+}
```

Comment:
this seems unrelated?

Reply:
For further debug, I believe we need it soon ...